lib/chef/node_map.rb

```diff
@@ -222,8 +219,12 @@ def compare_matchers(key, new_matcher, matcher)
         # Sort by class name (ascending) as well, if all other properties
         # are exactly equal
         # XXX: remove this in Chef-14 and use last-writer-wins (prepend if they match)
```

Comment:
Is this comment still valid?

Reply:
yes, there's an issue i cut on that one as well.   it blocks us from being able to trivially override core in cookbooks.